data/json/items/tool/electronics.json

```diff
@@ -347,6 +347,13 @@
       "EBOOKREAD",
       "ELECTRICSTORAGE",
       "PORTABLE_GAME",
+      {
+        "type": "effect_on_conditions",
+        "menu_text": "Dig into local files and logs",
```

Comment:
The only issue that i can't fix is that effect on condition action do not use menu_text
![image](https://user-images.githubusercontent.com/67688115/229286959-857542c9-7850-42b4-b719-322c4ab54c87.png)